src/implementations/config/helpers.ts

```diff
@@ -28,6 +28,10 @@ interface UpdatedForms {
   dnsTransferableRecord: Dns;
 }
 
+export const isV3Form = (formDefaults: OpenAttestationDocument): boolean => {
+  return "@context" in formDefaults && "openAttestationMetadata" in formDefaults;
+};
+
 export const getConfigWithUpdatedForms = ({
   configFile,
```

Comment:
this is almost the same as https://github.com/TradeTrust/tradetrust-config/blob/master/src/helpers/helpers.ts#L20 and https://github.com/TradeTrust/tradetrust-config/blob/master/src/helpers/helpers.ts#L83

should we move those helpers from tradetrust-config to here instead, then export out so we can share the same. wdyt

Reply:
ok